ios/FluentUI/Shimmer/MSFShimmerLinesView.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/**
+ Specialized ShimmerView that shows 1 or more shimmering lines.
+ */
+@objc(MSFShimmerLinesView)
+open class MSFShimmerLinesView: MSFShimmerView {
```

Comment:
You don't need the @objc on a separate line now, so you should be able to do just `@objc open class MSFShimmerLinesView:MSFShimmerView {` all in one go. Same in a few other place.